Unity/opentok/Podfile

```diff
@@ -0,0 +1,21 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+workspace 'holo.xcworkspace'
+
+target 'holo' do
+  platform :ios, '13.0'
+  project 'holo/holo.xcodeproj'
+  use_frameworks! :linkage => :static
+  # VonageMLTransformers 3.1.0-dev.345 installs WebRTC 99.10.0-dev.354
```

Comment:
> why do we need ml-transformers and not just onewebrtc?

In order to be able to use WebRTC from the capturer and renderer I had to remove single object prelinking in the video client SDK so now the app has to bring WebRTC and ML Transformers. That's way you see both there (ML Transformers is needed because the OpenTok library will need it by the time the app links).

> with the current pod i get build error of 438 duplicate symbols for architecture arm64

That's expected. I feel you did not used the OpenTok XC framework build as explained at https://vonage.slack.com/archives/C03STN3QSQ5/p1705505096588189.

Reply:
> why do we need ml-transformers and not just onewebrtc?

Done, ML Transformers is removed.